src/views/tickets/rsvp.php

```diff
@@ -104,7 +104,7 @@
 					 */
 					do_action( 'event_tickets_rsvp_before_confirmation_fields', $tickets );
 					?>
-					<table>
+					<table id="tribe-tickets-table">
```

Comment:
Would using a class instead of an id be possible here? 

I do realize the table already contains elements with IDs, but I feel like we shouldn't assume there will be only one ticket form per page (even if that's all that happens 'out of the box'): themers/customizers should be able to add the ticket form to each entry in list view, if they want, and actually I believe some are already doing this.